lib/disasters.js

```diff
@@ -1,23 +1,31 @@
 "use strict";
-var log = require('../log'),
-  models = require('../models'),
+ var models = require('../models'),
```

Comment:
models was being used previously in the code for methods like cache disasters or getDisasters, where you query the cache with models.cache.findOne() etc